### PR TITLE
feat: build amd64/arm64 LLVM Docker images

### PR DESCRIPTION
**Pull request by kuisathaverat**
build amd64/amr64 LLVM Docker images

depends on https://github.com/elastic/golang-crossbuild/pull/197
related to https://github.com/elastic/golang-crossbuild/issues/153

### CODE REVIEW

File: .ci/llvm-apple.groovy [hunk 1]
```diff
@@ -96,22 +94,14 @@ pipeline {
...
-  dir("${env.BASE_DIR}") {
-    retryWithSleep(retries: 3, seconds: 15, backoff: true) {
-      sh(label: "push docker image to ${env.REPOSITORY}", script: "make -C ${MAKEFILE} push")
+  withDockerEnv(secret: "${env.DOCKER_REGISTRY_SECRET}", registry: "${env.DOCKER_REGISTRY}") {
+    withGoEnv {
+      dir("${env.BASE_DIR}") {
+          retryWithSleep(retries: 3, seconds: 15, backoff: true) {
+            sh(label: 'Build Docker image', script: "make -C ${MAKEFILE} build")
+          }
+          sh(label: 'list Docker images', script: 'docker images --format "table {{.Repository}}:{{.Tag}}\t{{.Size}}" --filter=reference="docker.elastic.co/beats-dev"')
```
Comment (v1v) [inline .ci/llvm-apple.groovy:103]:
```suggestion
        retryWithSleep(retries: 3, seconds: 15, backoff: true) {
          sh(label: 'Build Docker image', script: "make -C ${MAKEFILE} build")
        }
      sh(label: 'list Docker images', script: 'docker images --format "table {{.Repository}}:{{.Tag}}\t{{.Size}}" --filter=reference="docker.elastic.co/beats-dev"')
```

**Review by v1v — COMMENTED** [review]
What's the reason to build without pushing the docker images?

**Comment by kuisathaverat** [comment]
>What's the reason to build without pushing the docker images?

buildx push them, see https://github.com/elastic/golang-crossbuild/issues/153

**Review by v1v — APPROVED** [review]

File: go/llvm-apple/Makefile [hunk 2]
```diff
@@ -3,13 +3,20 @@ include ../Makefile.common
 SUFFIX         := $(shell basename $(CURDIR))
 TAG 		   := $(REPOSITORY)/$(NAME):$(SUFFIX)$(TAG_EXTENSION)
 
+ifeq ($(DOCKER_MULTIARCH),1)
+DOCKER_CMD := $(SELF_DIR)/../.ci/scripts/buildx.sh
+
+push:
+	@echo "Already done."
```
Comment (kuisathaverat) [inline go/llvm-apple/Makefile:10]:
```suggestion
	@echo "Already done by buildx (.ci/scripts/buildx.sh)."
```

File: go/llvm-apple/Makefile [hunk 3]
```diff
@@ -3,13 +3,20 @@ include ../Makefile.common
 SUFFIX         := $(shell basename $(CURDIR))
 TAG 		   := $(REPOSITORY)/$(NAME):$(SUFFIX)$(TAG_EXTENSION)
 
+ifeq ($(DOCKER_MULTIARCH),1)
+DOCKER_CMD := $(SELF_DIR)/../.ci/scripts/buildx.sh
+
+push:
+	@echo "Already done by buildx (.ci/scripts/buildx.sh)."
```
Comment (kuisathaverat) [inline go/llvm-apple/Makefile:10]:
```suggestion
```suggestion
	@echo "Already done by buildx (.ci/scripts/buildx.sh)."
atomic-push:
	@echo "Already done by buildx (.ci/scripts/buildx.sh)."
```
```

**Pull request merged**